src/main/java/seedu/address/model/PackingListManager.java

```diff
@@ -64,7 +64,7 @@ public void resetDataPackingList(ReadOnlyPackingListManager newData) {
      * @param packingList the packing list
      */
     public void setPackingList(List<Item> packingList) {
-        this.packingList.setPackingList(packingList);
+        this.packingList.setPackingLists(packingList);
```

Comment:
See `line 131` of `UniquePackingList.java`

Either
```suggestion
        this.packingList.setItems(packingList);
```
Or
```suggestion
        this.packingList.setPackingList(packingList);
```